src/types/webhooks.ts

```diff
@@ -182,40 +182,81 @@ export type WebhookMessage = {
 	 * */
 	image?: WebhookMedia;
 	/**
-	 * When a customer selected a button or list reply.
+	 * Included when a customer interacts with an interactive message (button, list, or flow).
 	 */
 	interactive?: {
-		type: string;
 		/**
-		 * Sent when a customer clicks a button
+		 * The type of interactive message that the customer replied to.
 		 */
-		button_reply?: {
-			/**
-			 *  Unique ID of a button
-			 */
-			id: string;
-			title: string;
-		};
-		/**
-		 *  Sent when a customer selects an item from a list
-		 */
-		list_reply?: {
-			/**
-			 * Unique ID of the selected list item
-			 */
-			id: string;
-			title: string;
-			description: string;
-		};
-		/**
-		 *  Sent when a user submits a flow
-		 */
-		nfm_reply?: {
-			body: string;
-			name: string;
-			response_json: string;
-		};
-	};
+		type: "button_reply" | "list_reply" | "nfm_reply";
```

Comment:
Does this provide automatic autocompletion based on the type param? Coz I've never seen this syntax before, I would always use the [GenerateMappedNever](https://github.com/MarcosNicolau/whatsapp-business-sdk/blob/main/src/types/utils.ts#L42-L44), this looks so much simpler and better 💪.

Reply:
Btw: you can merge the pr if this is the case.

Reply:
It actually does, as you may see here... 😁


![Screenshot 2024-08-12 at 23 15 30](https://github.com/user-attachments/assets/185c208a-641b-4bbe-9d3d-dec98196c707)

---

![Screenshot 2024-08-12 at 23 17 44](https://github.com/user-attachments/assets/b545b8b4-7288-478f-bede-75c4c2a53f51)